docs/modules/ROOT/pages/component-prometheus.adoc

```diff
@@ -0,0 +1,21 @@
+= Prometheus
+
+[abstract]
+The Couchbase Prometheus Server can be used to collect cloud-native metrics from Couchbase Servers.
+
+== Overview
+
+https://prometheus.io/[Prometheus^] is a leading open-source monitoring solution. As a https://www.cncf.io/projects/prometheus/[graduated project^] of the Cloud Native Computing Foundation, it has become the de facto standard for metrics collection and alert generation on cloud platforms.
+
+Monitoring Couchbase is a critical aspect of production use case. Couchbase exposes detailed metrics that can be collected for visualization and other forms of monitoring. Prometheus fits perfectly as a monitoring solution for these exposed metrics to develop alerts for system issues, identify maintenance windows and other valuable business intelligence insights.
+
+image::component-prometheus.png[]
+
+== Get Started
+
+The Couchbase Prometheus Server is managed by the xref:architecture.adoc[Couchbase Monitoring Observability Stack]. Prometheus server is enabled by default as part of CMOS and it can be configured to monitor Couchbase cluster both for a Kubernetes based deployment done via Couchbase Operator Docs or for a Virtual Machine based deployment done via docker containers.
```

Comment:
I think I'd reword this to be the CMOS managed Prometheus server.

Also it says "Couchbase Operator Docs", is that supposed to be a link to docs? 

This all works for non containers too.

Reply:
Do we have documentation for non container based deployment? I would rather not bring it up unless we have them.

Reply:
>Also it says "Couchbase Operator Docs", is that supposed to be a link to docs?
Nice catch, will add link to operator page

Reply:
Ah I see what you're saying and it confused me so think it needs rewording slightly.

"or for a Virtual Machine based deployment done via docker containers" I thought you were talking about Couchbase Server here as you include it in the sentence about Couchbase Cluster which is the Server. Couchbase Server can be deployed via native binaries, containers and in Kubernetes. 

CMOS has to support all of those options and will be explicitly targeting the on-premise options (i.e. native and containers) for 0.1. CMOS itself will only support being deployed as a container.

I think this again is the confusion from the "Couchbase Prometheus Server", we need to be clear are we talking about CMOS or Server?

Additionally the inclusion of VM does not add anything here. You can do the same thing on a non-VM, by definition just a (bare metal) machine.

Reply:
So, does this sound better, `or for a on-premise deployment done via docker containers`? But again the deployment can be in cloud and not on premise.

Reply:
I think this section needs a bit of rewording, it's confusing CMOS deployment and Couchbase Server deployment.

It starts talking about CMOS enabled Prometheus then moves to Couchbase Cluster (i.e. Couchbase server) deployed via CAO or on-premise then goes back to talking about container deployment of CMOS all in the same sentence.

Something like:
"CMOS provides a Prometheus server internally that is enabled by default.
CMOS can be configured to scrape Couchbase clusters provided either as on-premise deployments (bare metal or VM) or Kubernetes deployments managed by the Couchbase Autonomous Operator.
CMOS itself is deployed as a container in both types of infrastructure, i.e. either running on a host as a manually orchestrated container or within Kubernetes as a pod."

Reply:
Explicitly we support Couchbase Server running as a native binary, i.e. not as a container. We also support it running as a container.

We only ever support CMOS running as a container for all use cases. This is what confused me initially and will confuse others so I think it needs to be clearer and shorter sentences.

Reply:
Since we are writing a page about cloud native database, I have tried to include some info about both Couchbase server prometheus and CMOS prometheus.

Reply:
That's fine just make it clear which is being used I think. As I say, I designed this stack and it confused me when reading it :-)